### PR TITLE
[14.0][FIX] fixed multicurrency scenario

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
Why not using amount_residual_currency?

**Comment by ChrisOForgeFlow** [comment]
> 

This field exists on account_move_line but in this case needed header value, because that conversion is performed again

**Review by AaronHForgeFlow — APPROVED** [review]
Yes, my bad, I though it was also in the account.move.

Code review LGTM! :+1:

File: account_payment_term_discount/wizard/account_payment_register.py [hunk 1]
```diff
@@ -44,7 +44,14 @@ def onchange_payment_amount(self):
                 )
                 payment_date = fields.Date.from_string(self.payment_date)
                 discount_amt = self.invoice_id.discount_amt
-
+                amount_residual = self.invoice_id.amount_residual
+                if self.invoice_id.currency_id.id != self.currency_id.id:
+                    discount_amt = self.invoice_id.currency_id.compute(
```
Comment (MiquelRForgeFlow) [inline account_payment_term_discount/wizard/account_payment_register.py]: warning: The `compute` method is deprecated. Use `_convert` instead

the same for the other.

**Review by MiquelRForgeFlow — COMMENTED** [review]
also add another commit to delete the useless few `from_string` from lines above.

File: account_payment_term_discount/wizard/account_payment_register.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply account_payment_term_discount/wizard/account_payment_register.py]: Thanks!

File: account_payment_term_discount/wizard/account_payment_register.py [hunk 2]
```diff
@@ -44,7 +44,20 @@ def onchange_payment_amount(self):
                 )
                 payment_date = fields.Date.from_string(self.payment_date)
```
Comment (MiquelRForgeFlow) [inline account_payment_term_discount/wizard/account_payment_register.py]:
```suggestion
                payment_date = self.payment_date
```

Comment (MiquelRForgeFlow) [reply account_payment_term_discount/wizard/account_payment_register.py]: same in line 41

Comment (ChrisOForgeFlow) [reply account_payment_term_discount/wizard/account_payment_register.py]: Done!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request closed without merging**